x/evmutil/keeper/conversion_evm_native.go

```diff
@@ -222,3 +239,19 @@ func (k Keeper) LockERC20Tokens(
 
 	return err
 }
+
+// MulByPowersTen computes i*10^power and returns integers for the part before and after the decimal point.
+func MulByPowersTen(i *big.Int, power int64) (*big.Int, *big.Int) {
+	positivePower := big.NewInt(power)
```

Comment:
do we need to do some validation of inputs here, return an error if `i` or `power` is outside of the expected range?

Reply:
yeah validation would be good, not sure what the limits of `big.Int`s are, but there should be at least some validation in the `decimal_conversion` param because we'll never be converting from like 1 decimal place to 1000 places.